src/pages/Main.js

```diff
@@ -32,13 +32,21 @@ const Main = () => {
       h(ImportStatus),
       h(Favicon),
       h(IdleStatusMonitor),
-      h(ErrorWrapper, [
-        h(TitleManager),
-        h(FirecloudNotification),
-        h(AuthenticatedCookieSetter),
-        h(AuthProvider, getOidcConfig(), [h(AuthStoreSetter)]),
-        h(AuthContainer, [h(Router)]),
-      ]),
+      h(
+        ErrorBoundary,
+        {
+          onError: (error) => {
```

Comment:
because we are adding some new code here, can we add unit tests for it?  And Main.js looks fairly small, can we try converting to TS while we are here?

Reply:
I can try TS conversion.

I'm not sure what unit tests for this would look like. I could mock all these components and check that they're rendered, but I don't see much value there.

Reply:
that would be good... it may feel a bit meh... but unit testing that a composer is "composing" the right things is a good link the chain to have.

Reply:
Added some tests in https://github.com/DataBiosphere/terra-ui/pull/4247/commits/bf7455912ad2e080bc2bee54540093bb3b2805ed. So many mocks...